prow/cmd/deck/main.go

```diff
@@ -124,6 +124,11 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
+	if !o.runLocal {
+		// in the container we place static here
+		staticFilesLocation = "/static"
```

Comment:
Why not runtime configure this?

Reply:
/shrug
The location is fixed, previously these were inline to registering the handlers and we did not have a global for this. I would prefer to not have the global, but apparently we're ok with that? /shrug

Reply:
/shrug

Reply:
I'm fine with a different PR, I've been in meetings so I just made a quick fix to get things moving forward, it seemed we moved this value from a local to a global for some reason previously glancing through the history. 

I went ahead and deployed this since apparently deck had been 502ing for 2+ hours >.>

Reply:
it is a flag now